### PR TITLE
*: Support show current background job info

### PR DESCRIPTION
**Pull request by zimulala**
#800 
Handle to the last TODO.

### CODE REVIEW

File: ddl/stat.go [hunk 1]
```diff
@@ -44,42 +47,70 @@ func (d *ddl) GetScope(status string) variable.ScopeFlag {
 	return variable.DefaultScopeFlag
 }
 
-// Stat returns the DDL statistics.
-func (d *ddl) Stats() (map[string]interface{}, error) {
+func replacePrefix(str, old, new string) string {
```
Comment (qiuyesuifeng) [inline ddl/stat.go]: Seems needless to extract a function, it is not too long.

File: ddl/stat.go [hunk 2]
```diff
@@ -44,42 +47,70 @@ func (d *ddl) GetScope(status string) variable.ScopeFlag {
...
+	return nil
+}
+
+// Stat returns the DDL statistics.
+func (d *ddl) Stats() (map[string]interface{}, error) {
+	m := make(map[string]interface{})
+	m[ddlServerID] = d.uuid
+
+	if err := stats(d.store, m, ddlJobFlag); err != nil {
+		return nil, err
```
Comment (qiuyesuifeng) [inline ddl/stat.go]: errors.Trace(err)

File: ddl/stat.go [hunk 3]
```diff
@@ -44,42 +47,70 @@ func (d *ddl) GetScope(status string) variable.ScopeFlag {
...
-		m[ddlJobLastUpdateTS] = info.Job.LastUpdateTS / 1e9
-		m[ddlJobState] = info.Job.State.String()
-		m[ddlJobError] = info.Job.Error
-		m[ddlJobSchemaState] = info.Job.SchemaState.String()
-		m[ddlJobSchemaID] = info.Job.SchemaID
-		m[ddlJobTableID] = info.Job.TableID
-		m[ddlJobSnapshotVer] = info.Job.SnapshotVer
-		m[ddlJobReorgHandle] = info.ReorgHandle
-		m[ddlJobArgs] = info.Job.Args
+	m[replacePrefix(ddlJobID, ddlPrefix, prefix)] = info.Job.ID
```
Comment (qiuyesuifeng) [inline ddl/stat.go]: I think using buildXXXKey maybe better than using replace.

File: inspectkv/inspectkv.go [hunk 4]
```diff
@@ -65,6 +65,31 @@ func GetDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
...
+	}
+	info.SchemaVer, err = t.GetSchemaVersion()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	if info.Job == nil {
+		return info, nil
+	}
+
+	return info, nil
```
Comment (qiuyesuifeng) [inline inspectkv/inspectkv.go]: Seems can directly `return info, nil`?

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @coocood @shenli

**Comment by coocood** [comment]
LGTM

**Pull request merged**